### PR TITLE
Teams (part 2)

### PR DESCRIPTION
**Pull request by niwinz**
This PR contains:
- Improvements on how to interact with response from services.
- Team administration realted mutations and queries (invitation, tokens, emails, etc..)
- Some fixes
- User profile refactor
- Modals refactor

### CODE REVIEW

File: frontend/src/app/main/ui.cljs [hunk 1]
```diff
@@ -186,3 +188,9 @@
       (ts/schedule 100 #(st/emit! (dm/show {:content "Something wrong has happened."
                                             :type :error
                                             :timeout 5000}))))))
+
```
Comment (hirunatan) [inline frontend/src/app/main/ui.cljs:191]: delete this?

Comment (niwinz) [reply frontend/src/app/main/ui.cljs:191]: I'll do it in other MR, i want to research more on this.

Comment (hirunatan) [reply frontend/src/app/main/ui.cljs:191]: ok

File: frontend/src/app/main/ui/dashboard/sidebar.cljs [hunk 2]
```diff
@@ -158,123 +117,239 @@
...
+     [:div.modal-container.confirm-dialog
+      [:div.modal-header
+       [:div.modal-header-title
+        [:h2 "Before you leave"]]
+       [:div.modal-close-button
+        {:on-click on-cancel} i/close]]
+
+      [:div.modal-content.generic-form
+       [:p "You are " (:name team) " owner."]
+       [:p "Select an other member to promote before leave."]
```
Comment (hirunatan) [inline frontend/src/app/main/ui/dashboard/sidebar.cljs:190]: many untranslated texts, is this planned to update later?

**Pull request merged**